xbmc/pvr/PVRJobs.h

```diff
@@ -8,17 +8,19 @@
 
 #pragma once
 
-#include <memory>
-#include <vector>
-
-#include "addons/PVRClient.h"
 #include "addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h"
-#include "utils/JobManager.h"
+#include "threads/SystemClock.h"
+#include "utils/Job.h"
 
-#include "pvr/PVRTypes.h"
+#include <memory>
+#include <string>
+#include <vector>
 
 namespace PVR
 {
+  class CPVRChannelGroup;
+  class CPVRClient;
```

Comment:
do not intend namespaces, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#32-indentation